.github/runs-on.yml

```diff
@@ -5,3 +5,39 @@ images:
     owner: "405894839083"
     name: "arm64-rust-riscv-dev-32gb"
     ami: "ami-040a96edd406c9478"
+
+runners:
+  test-64cpu-arm64:
```

Comment:
none of this works since it's not on `main` branch, also I prefer for cpu we just use runson runner types